import_counts.py

```diff
@@ -0,0 +1,103 @@
+from os import path
+from datetime import datetime
+import json
+import boto.s3
+import boto.provider
+import postgres
+
+with open("config.json") as f:
+    CONFIG = json.loads(f.read())
+
+if "aws_access_key_id" not in CONFIG:
+    aws = boto.provider.Provider("aws")
+    CONFIG["aws_access_key_id"] = aws.get_access_key()
+    CONFIG["aws_secret_access_key"] = aws.get_secret_key()
+
+S3_REGION = "us-east-1"
+S3_BUCKET = "net-mozaws-prod-us-west-2-pipeline-analysis"
+S3_PREFIX = "fxa-basic-metrics/"
+S3_URI = "s3://" + S3_BUCKET + "/" + S3_PREFIX + "fxa-basic-metrics-{day}.txt"
+DB_URI = "postgresql://{db_username}:{db_password}@{db_host}:{db_port}/{db_name}".format(**CONFIG)
+COUNTS_BEGIN = datetime.strptime("2017-05-30", "%Y-%m-%d")
+
+Q_DROP_CSV_TABLE = "DROP TABLE IF EXISTS temporary_raw_counts;"
+
+Q_CREATE_COUNTS_TABLE = """
+    CREATE TABLE IF NOT EXISTS counts (
+      day DATE NOT NULL UNIQUE SORTKEY ENCODE RAW,
+      accounts BIGINT NOT NULL ENCODE ZSTD,
+      verified_accounts BIGINT NOT NULL ENCODE ZSTD
+    );
+"""
+
+Q_CHECK_FOR_DAY = """
+    SELECT day FROM counts
+    WHERE day = '{day}';
+"""
+
+Q_CREATE_CSV_TABLE = """
+    CREATE TABLE IF NOT EXISTS temporary_raw_counts (
+      day CHAR(10) NOT NULL UNIQUE SORTKEY,
```

Comment:
...although perhaps I should also have made this `VARCHAR(11)` so that we don't need to change the script in 8,000 years time. :smile: